app/android/app/google-services.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
shit! need to rotate the credentials :/

Reply:
maybe we don't need to rotate as the information is not critical. but I pinged firebase support